src/components/ValidatedInput/ValidFieldText.tsx

```diff
@@ -26,31 +26,39 @@ export class Input extends FieldText {
   }
 }
 
-interface Props extends React.InputHTMLAttributes<HTMLInputElement> {
+export interface ValidFieldTextProps extends React.InputHTMLAttributes<HTMLInputElement> {
   onKeyDown?: (event: React.FormEvent<HTMLInputElement>) => void;
   label: string;
   name: string;
   value: string;
   errMessage?: string;
-  onChange: (event: React.FormEvent<HTMLInputElement>) => void;
+  onChange?: any;
```

Comment:
any reason why change this to `any`?

Reply:
I changes this base on the current implementation in admin portal, after I've check it again mostly the `onChange` props implementation is using `React.ChangeEvent<HTMLInputElement>`, I will update this component to use the ChangeEvent type.